src/plugins/formvalid/formvalid-server-fr.hbs

```diff
@@ -28,7 +28,7 @@
 
 	{{>alertariahidden}}
 
-	<p>Après avoir soumis votre page, le serveur retoune 4 erreurs serveur. Maintenant si vous essayez de resoumettre
+	<p>Après avoir soumis votre page, le serveur retoune 6 erreurs serveur. Maintenant si vous essayez de resoumettre
```

Comment:
Added example so we need to increase the number.

```suggestion
	<p>Après avoir soumis votre page, le serveur retourne 7 erreurs serveur. Maintenant si vous essayez de resoumettre
```

Reply:
Count updated.